app/controllers/v1/activities_controller.rb

```diff
@@ -18,14 +18,24 @@ def generate_alias
     render json: alias_response("#{mail_alias}@csvalpha.nl")
   end
 
-  def ical
+  def ical # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     return head :unauthorized unless authenticate_user_by_ical_secret_key
 
-    permitted_categories = (params[:categories].try(:split, ',') & Activity.categories) ||
+    requested_categories = params[:categories].try(:split, ',')
+
+    permitted_categories = (requested_categories & Activity.categories) ||
```

Comment:
did this really have to be split into separate variables?

Reply:
Not necessarily but I found it more readable than everything on one line